worlds/ladx/Items.py

```diff
@@ -305,3 +305,120 @@ class ItemName:
 links_awakening_items_by_name = {
     item.item_name : item for item in links_awakening_items
 }
+
+links_awakening_item_name_groups = {

```

Comment:
```suggestion
links_awakening_item_name_groups: Dict[str, Set[str]] = {
```
And also import `Dict` and `Set` from `typing`